app/assets/stylesheets/components/_disclaimer.scss

```diff
@@ -2,6 +2,7 @@
   background-color: $navy;
   color: $white;
   font-size: $h6;
+  font-weight: lighter;
```

Comment:
can you call this `300`? i kinda prefer the more explicitness and i think those are equivalent in this case